1.9/administration/debugging/quickstart.md

```diff
@@ -0,0 +1,194 @@
+
+1.  Deploy and run a job with the DC/OS CLI:
+
+    1.  Create the following job definition and save as `⁠⁠⁠⁠test-job.json`. This specifies a sleep job that runs for `10000000` seconds.
+    
+        ```bash
+        {
+            "id": "task-exec-test",
+            "cmd": "sleep 999999999",
+            "cpus": 0.1,
+            "instances": 1
+        }
+        ```
+    
+    1.  Deploy the job with this CLI command:
+    
+        ```bash
+        $ dcos job add test-job.json
+        ```
+        
+    1.  Verify that the job has been successfully deployed:
+    
+        ```bash
+        $ dcos job list
+        ID              DESCRIPTION  STATUS   LAST SUCCESFUL RUN        
+        task-exec-test               Unscheduled         None
+        ```
+
+    1.  Run the job:
+    
+        ```bash
+        $ dcos job run task-exec-test
+        ```
+
+1.  Get the task ID of the job with this CLI command:
+
+    ```bash
+    $ dcos task
+    ```
+    
+    The output should look similar to this: 
+    
+    ```bash
+    NAME                                HOST       USER  STATE  ID                                                                       
+    20161209183121nz2F5.task-exec-test    10.0.2.53  root    R    <task_id>
+    ```
+
+1.  Launch a process inside of the container with the task ID (`<task_id>`) specified and attach a TTY to it. This will launch an interactive Bash session.
+
+    ```bash
+    $ dcos task exec --interactive --tty <task_id> bash
+    ```
+    
+    You should now be inside the container running an interactive Bash session.
+    
+    ```bash
+    root@ip-10-0-2-53 / #
+    ```
+    
+    **Tip:** You can use shorthand abbreviations `-i` for `--interactive` or `-t` for `--tty`. Also, only the beginning unique characters of the `<task_id>` are required. For example, if your task ID is `exec-test_20161214195` and there are no other task IDs that begin with the letter `e`, this is valid command syntax: `dcos task exec -i -t e bash`. For more information, see the CLI command [reference](/docs/1.9/usage/cli/command-reference/).
+    
+1.  Run a command from the interactive Bash session. For example, the `ls` command:
+
+    ```bash
+    root@ip-10-0-1-104 / # ls
+    bin   dev  home  lib64	     media  opt   root	sbin  sys  usr
+    boot  etc  lib	 lost+found  mnt    proc  run	srv   tmp  var
+    ```
+
+# Pipe output from a command running inside a container 
+
+You can run commands inside a container by using the `dcos task exec` command. In this example, a long running Marathon app is launched and then the `dcos task exec` command is used to get the hostname of the node running the app. 
+
+1.  Create a Marathon app definition and name it `my-app.json` with the following contents:
+
+    ```bash
+    {
+       "id": "/my-app",
+       "cmd": "sleep 100000000",
+       "cpus": 1,
+       "instances": 1
+     }
+     ```
```

Comment:
Do we maybe want to rename this app to something else? If someone runs through these instructions from top to bottom, this app-id will conflict with the one already deployed by the first set of instructions and they won't be able to launch the app.

Also, I think this should probably be the first example in this document. I think it flows better from least complicated (stream output only) to most complicated (full interactive bash session).

Finally, This app definition (other than the `id`) should probably match the other one for consistency.